### PR TITLE
Add option to flip video

### PR DESCRIPTION
**Pull request by bahar-ap**
## Description

Adding option to flip video

## Screenshots

mosquitto_sub -t '#' -v -h localhost
OR 
mosquitto_sub -t 'camera/flip_video_feed' -v -h localhost

when clicking flip video, mqtt message should be sent :)

### CODE REVIEW

File: client/src/api/common/powerModel.ts
Comment (kdav108) [inline client/src/api/common/powerModel.ts]: I get tripped with this every time, but pretty sure the spelling was correct before, i.e. it's `Achieved`.

File: client/src/components/common/camera_settings/OverlaySelection.tsx
Comment (kdav108) [inline client/src/components/common/camera_settings/OverlaySelection.tsx]: Glad to know 😂

File: client/src/components/common/camera_settings/OverlaySelection.tsx
Comment (kdav108) [inline client/src/components/common/camera_settings/OverlaySelection.tsx]: Maybe make this toast appear after the `emit` call, so if that errors out we don't incorrectly send a success toast message.

**Review by kdav108 — CHANGES_REQUESTED** [review]
Nice short PR! Nothing to complain here, just a few nit picks 😂.

Also, it seems you've branched off from `bahar-ap/change_achieved_max_speed_name` instead of `master`, that's fine, just make sure you merge #135 first and then update this branch before merging this :)

File: client/src/components/common/camera_settings/OverlaySelection.tsx
Comment (bahar-ap) [reply client/src/components/common/camera_settings/OverlaySelection.tsx]: whoops!!! 😅

File: server/js/sockets.js
Comment (kdav108) [inline server/js/sockets.js:379]: You forgot to remove this as well 😂

File: client/src/components/v3/StatisticRow.tsx
Comment (kdav108) [inline client/src/components/v3/StatisticRow.tsx:30]: we don't require this `ESLint-disable-next-line` stuff cos we now do use the `setPrevTrapSpeed` func. Previously we had it defined but never called it and hence we got the `no-unused-vars` error from ESLint, and had to disable it.

File: client/src/components/v3/StatisticRow.tsx
Comment (kdav108) [inline client/src/components/v3/StatisticRow.tsx:39]: Same here, should not be needed anymore

**Review by kdav108 — APPROVED** [review]
We're done, just really minor stuff to get rid of.

I'll approve still, we can merge after the comments below are resolved.

**Review by hallgchris — APPROVED** [review]
Nothing to add! In the future it would be great if you could include a screenshot in the PR description!

**Pull request merged**
